oat-sa/extension-tao-testqti
# Fix/ADF-1789/UI glitches

### Related to: https://oat-sa.atlassian.net/browse/ADF-1789

### Summary

Fix a few UI glitches with the translation properties:
- the tooltip was not installed
- the scope was confusing as it was not clear if the props apply to the whole test or to the section/rubric

![image](https://github.com/user-attachments/assets/a4eddd0c-be96-45d4-99e7-34d4c75e9f8f)
![image](https://github.com/user-attachments/assets/a4347480-1291-4363-991a-733c04217ccb)

### How to test

- check out the branch: `git checkout -t origin/fix/ADF-1789/ui-glitches`
- make sure to have the companion branch checked out and up to date too on the other extensions: `feat/HKD-6/integration`
- make sure to have the Feature Flags activated
```
FEATURE_FLAG_TRANSLATION_ENABLED=true
FEATURE_FLAG_UNIQUE_NUMERIC_QTI_IDENTIFIER=true
```
- open a test for translation
- check the translation status
- open the props of any part/section/item/rubric
